openlibrary/plugins/openlibrary/js/autocomplete.js

```diff
@@ -1,7 +1,7 @@
 // jquery plugins to provide author and language autocompletes.
 
-;(function($) {
```

Comment:
these ;'s need to be here because we are concat'ing files together

Reply:
Weird. This should have an eslint ignore above the line or ignored in linting... what happened to break this?

(Note it will autofix when this is not the case so not @dnguneratne fault here!)

I'll have a look at the git history this week as it's possible some old branches got merged and regressed a few things..